src/EnergyPlus/WeatherManager.hh

```diff
@@ -198,6 +198,10 @@ namespace WeatherManager {
         bool ActualWeather;              // true when using actual weather data
         int RawSimDays;                  // number of basic sim days.
         bool firstHrInterpUseHr1;        // true when using Hour 1 for first hour interpolations; false to use Hour 24
+        Real64 maxCoolingOATSizing;      // max outdoor dry-bulb for DesignDay or RunPeriodDesign type weather
+        Real64 maxCoolingOADPSizing;     // outdoor dew point at max outdoor dry-bulb for DesignDay or RunPeriodDesign type weather
+        Real64 minHeatingOATSizing;      // min outdoor dry-bulb for DesignDay or RunPeriodDesign type weather
+        Real64 minHeatingOADPSizing;     // outdoor dew point at min outdoor dry-bulb for DesignDay or RunPeriodDesign type weather
```

Comment:
When we add new members to classes in the future, just go ahead and initialize them at the declaration:

```
Real64 maxCoolingOATSizing = -999.0;
```

And preferably, not with -999 because that's a magic number.  This is OK for this PR.